synapse-get-annotations-tool.cwl

```diff
@@ -0,0 +1,53 @@
+
+#!/usr/bin/env cwl-runner
+#
+# Authors: Andrew Lamb
```

Comment:
please add `dct:creator` tags in the get and set annotations (or schema.org `Person` as in the [CWL user guide](https://www.commonwl.org/user_guide/17-metadata/index.html)).

Reply:
I was having trouble using the old format I had seen in some CWL using `dct:creator`. The schema.org `Person` works for me.